Pipfile

```diff
@@ -1,12 +0,0 @@
-[[source]]
```

Comment:
Wouldn't this make the package unusable with pipenv only?

Reply:
pipenv support removed. Do everything with poetry. Same as bx_py_utils